scripts/export/convert_nemo2_for_export.py

```diff
@@ -1,123 +0,0 @@
-# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
We should still keep this but maybe rename it `convert_nemo2_to_nemo1` or something. Wdyt?

Reply:
Fixed in latest commit.